WordPressShared/Core/Utility/NSDate+Helpers.swift

```diff
@@ -127,17 +145,19 @@ extension Date {
     /// - Example: 2 days ago
     /// - Example: Jan 22, 2017
     ///
-    public func mediumString(timeZone: TimeZone? = nil) -> String {
-        let relativeFormatter = TTTTimeIntervalFormatter()
+    public func toMediumString(inTimeZone timeZone: TimeZone? = nil) -> String {
+        let relativeFormatter = RelativeDateTimeFormatter()
+        relativeFormatter.dateTimeStyle = .named
+
         let absoluteFormatter = DateFormatters.mediumDate
-        
+
         if let timeZone = timeZone {
             absoluteFormatter.timeZone = timeZone
```

Comment:
~Even if `RelativeDateTimeFormatter` doesn't have a `timeZone` property, it has a `calendar` property; so shouldn't we set the `relativeFormatter.calendar.timeZone = timeZone` too here?~

nvm, tested it in the Swift REPL and it seems that for RelativeDateTimeFormatter the tz doesn't really matter. Which makes kinda sense after all, since 10 minutes ago is 10 minutes ago in any timezones, since it is formatting a duration / number of seconds between `self` and "now", so those are "two moments in time in the universe" unrelated to timezones…